src/pages/blog/Blog.js

```diff
@@ -1,7 +1,15 @@
-function Blog() {
+import ComingSoon from '../../components/menu/ComingSoon/ComingSoon.js'
+
+
+function Blog ()
+{  
   return (
-    <div>Interesting articles</div>
+    <div>
+     <ComingSoon/>         
+     This blog page will include interesting articles.
```

Comment:
I think these descriptions can be removed - it's pretty obvious what a blog page will contain.
```suggestion
```